mizar/obj/networkpolicy.py

```diff
@@ -26,28 +26,16 @@
 
 
 class NetworkPolicy:
-    def __init__(self, name, obj_api, opr_store, spec=None):
```

Comment:
Please consider add network policy's metadata / spec in NetworkPolicy object in the future. Seems like you are only using NetworkPolicy object to store list of endpoints associated with the Network policy. If so, you can just use operator store to store this information. Seems pointless to have network policy object

Reply:
Good idea. After planned "issues" done, will revisit this.